Created VM Insights table. (#186)
Azure/alz-monitor: docs/wiki/AlertsDetails.md

| [[DINE] Deploy Activity Log VPN Gateway Delete Alert](https://github.com/Azure/alz-monitor/blob/main/src/resources/Microsoft.Authorization/policyDefinitions/deploy-activitylog-VPNGate-Del.bicep)                | ActivityVPNGatewayDelete    | managementGroup | Administrative | Microsoft.Network/vpnGateways/delete                                | succeeded | Subscription | No          | [Activity Log Service Notifications](https://learn.microsoft.com/en-us/azure/service-health/alerts-activity-log-service-notifications-portal)<br>[Best practices for setting up service health alerts](https://www.microsoft.com/en-us/videoplayer/embed/RE2OtUa) |

## VM Insights Log Alerts

> Once VM Insights has been onboard into your environment the following alert rules can be deployed.

Reviewer: ```suggestion
Once VM Insights has been enabled in your environment, the following alert rules can be configured for use via this the Baseline Alerts framework.
```

N/A: Not applicable, not used in the query or used as a parameter.

> **IMPORTANT!**
>

> > The "\\|" has been added to each KQL query so that they can be included in the table properly. Remove the "\\" if you're going to run these

Reviewer 2: ```suggestion
> The "\\" has been added before the "\|" in each KQL query so that they can be included in the table properly. Remove the "\\" if you're going to run these
```

Author: I think I'll remove it altogether, if someone were to copy and paste the KQL while viewing the table in their browser it's fine. It only looks weird if you're looking at it in markdown.

Author: Removed the note, it's not applicable.

| [[DINE] Deploy Activity Log VPN Gateway Delete Alert](https://github.com/Azure/alz-monitor/blob/main/src/resources/Microsoft.Authorization/policyDefinitions/deploy-activitylog-VPNGate-Del.bicep)                | ActivityVPNGatewayDelete    | managementGroup | Administrative | Microsoft.Network/vpnGateways/delete                                | succeeded | Subscription | No          | [Activity Log Service Notifications](https://learn.microsoft.com/en-us/azure/service-health/alerts-activity-log-service-notifications-portal)<br>[Best practices for setting up service health alerts](https://www.microsoft.com/en-us/videoplayer/embed/RE2OtUa) |

## VM Insights Log Alerts

> Once VM Insights has been enabled in your environment, the following alert rules can be configured for use via this the Baseline Alerts framework.

Reviewer 3: ```suggestion
Once VM Insights has been enabled in your environment, the following alert rules can be configured for use via the Baseline Alerts framework.
```

Author: Makes sense to me.

> N/A: Not applicable, not used in the query or used as a parameter.

Reviewer 3: What's this for?

Author: Some times the EvaluationPeriods and FailingPeriods aren't used in the alert, in which case they are Not Applicable.

Author: Sometimes ComputersToInclude and OtherResources isn't needed or applicable.
